From a197a689437f9619615a7850a3d4af3d30a47df7 Mon Sep 17 00:00:00 2001
From: Xing Zhao <xzhao@cavium.com>
Date: Tue, 13 Mar 2012 15:43:08 -0700
Subject: [PATCH 167/337] MIPS: Change sparsemem physical memory bits from 35
 to 38

Based On SDK 3.0.0-482

Some systems have large amounts of RAM and require this.

Signed-off-by: Xing Zhao <xzhao@cavium.com>
Signed-off-by: David Daney <david.daney@cavium.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
Signed-off-by: Jack Tan <jack.tan@windriver.com>
---
 arch/mips/include/asm/sparsemem.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/mips/include/asm/sparsemem.h b/arch/mips/include/asm/sparsemem.h
index 4461198..cee4f1c 100644
--- a/arch/mips/include/asm/sparsemem.h
+++ b/arch/mips/include/asm/sparsemem.h
@@ -11,7 +11,7 @@
 #else
 # define SECTION_SIZE_BITS	28
 #endif
-#define MAX_PHYSMEM_BITS        35
+#define MAX_PHYSMEM_BITS        38
 
 #endif /* CONFIG_SPARSEMEM */
 #endif /* _MIPS_SPARSEMEM_H */
-- 
1.7.5.4

